addons/website/static/src/snippets/s_announcement_scroll/announcement_scroll.edit.scss

```diff
@@ -0,0 +1,22 @@
+// Snippets Modal Preview
+// ======================
+// Performs an animation on :hover showcasing the user how the snippet behaves
+// in use. Measurements and values are arbitrary, the goal is just to provide
+// a preview.
+.o_snippet_preview_wrap {
+    &:hover [data-snippet="s_announcement_scroll"] {
+        .s_announcement_scroll_marquee_item {
+            animation: o-marquee-scroll-left-infinite-preview linear infinite 10s;
+        }
+        animation-play-state: running !important;
+    }
+
+    @keyframes o-marquee-scroll-left-infinite-preview {
+        0% {
+            transform: translateX(0%);
+        }
+        100% {
+            transform: translateX(-100%);
+        }
+    }
```

Comment:
Do we need a "preview" version of the animation? Can't we use ` o-marquee-scroll-left-infinite` direclty?